### DISCUSSION TITLE
Can i safely delete: claimdata, logs and playerdata if i want to reset the claims etc?

### DISCUSSION THREAD

**Discussion by Rikk010** [GriefPrevention/GriefPrevention#845] [Q&A] [answered]
The answer to your question may exist in the wiki, go check that out first! 

Wiki: https://github.com/TechFortress/GriefPrevention/wiki

Otherwise, if you have a spigot account, you may have better luck posting in the discussion thread so people who follow the thread may see your question. https://www.spigotmc.org/threads/griefprevention.35615/page-311

Still no luck? Really? Then go ahead and ask here.

**Comment by RoboMWM** [comment, marked as answer]
yes.
